pkg/controller/kieapp/deploy_ui.go

```diff
@@ -0,0 +1,167 @@
+package kieapp
+
+import (
+	"context"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/constants"
+	routev1 "github.com/openshift/api/route/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+var name = "console-cr-form"
+var app = "kie-cloud-operator"
```

Comment:
better to use `opName` which is set and passed back from `UseEmbeddedFiles` as seen here:
https://github.com/kiegroup/kie-cloud-operator/blob/6478199f839626fd9a68bba11431b5e5a992cb8f/pkg/controller/kieapp/defaults/defaults.go#L582

this can be passed into `deployConsole` method as new input instead. This is preferable because our product release name is actually business-automation-operator but its set dynamically.